custom_components/eo_mini/api.py

```diff
@@ -79,6 +79,13 @@ async def async_get_session(self) -> list[dict]:
         "Get the current session if any"
         return await self._async_api_wrapper("get", f"{self.base_url}/api/session")
 
+    async def async_get_session_liveness(self) -> bool:
+        "Get the session liveness state"
```

Comment:
Perhaps explain what liveness means instead of repeating the function name in the docstring :)

Reply:
Good idea! I work with Kubernetes on a daily basis where 'liveness' is a very common term. It makes sense to explain it here, thanks for pointing this out. Will update!

Reply:
Pushed an updated for this to be more descriptive